plugins/verifier/sample/sample.go

```diff
@@ -37,7 +38,14 @@ type PluginInputConfig struct {
 }
 
 func main() {
+	// send info and debug messages to stderr
+	fmt.Fprintln(os.Stderr, "info: initialized plugin")
```

Comment:
Wondering if the following statement is true?  "stdOut should only be used for VerifierReport response, all informational should be written to stderr". What is the best place to document this

Reply:
It's not entirely true. 😅I wrote the feature to handle multiple messages form stdout and stderr that way plugin developers could write to either. The comment is there to explain what the following 2 lines are doing, so maybe it's more accurate to say "sends info and debug messages to stderr" I could also add that it's acceptable to also write to stdout. 

I'm open to suggestions though.

Reply:
@duffney does this mean the user could do something like this: `fmt.Fprintln(os.Stdout, "info: initialized plugin")` and it would work equivalently?

Reply:
Currently only stderr adds the extra information to stderr messages. But that makes the assumption that people always put those messages there and no in stdout, which isn't always the case. 

I'll start looking at the ratify logger to working on use that to display the error messages.
 
---

QQ: is it important to distinguish between info, debug, and warn etc? 

If not, then I can just return the message sent by the plugin with the ratify logger and remove the switch that adds the log prefix.

Reply:
I think it would be nice if we could. We have issues already where the logs are just too verbose and being able to only display info vs debug will be very helpful.